modules/core/src/main/java/org/apache/ignite/internal/processors/cache/IgniteCacheOffheapManager.java

```diff
@@ -578,6 +578,11 @@ public long cacheEntriesCount(int cacheId, boolean primary, boolean backup, Affi
          * @return Old row.
          */
         @Nullable public CacheDataRow oldRow();
+
+        /**
+         * Flag that indicates if oldRow was expired during invoke
+         */
```

Comment:
```suggestion
         * Flag that indicates if oldRow was expired during invoke.
         * @return {@code true} if old row was expired, {@code false} otherwise.
         */
```